src/drivers/uavcannode/UavcanNode.cpp

```diff
@@ -352,6 +354,32 @@ void UavcanNode::Run()
 		}
 	}
 
+	// differential_pressure -> uavcan::equipment::air_data::RawAirData
+	if (_diff_pressure_sub.updated()) {
+		differential_pressure_s diff_press;
+
+		if (_diff_pressure_sub.copy(&diff_press)) {
+
+			uavcan::equipment::air_data::RawAirData raw_air_data{};
+
+			// raw_air_data.static_pressure =
+			raw_air_data.differential_pressure = diff_press.differential_pressure_raw_pa;
+			// raw_air_data.static_pressure_sensor_temperature =
+			raw_air_data.differential_pressure_sensor_temperature = diff_press.temperature;
+			// raw_air_data.static_air_temperature
+			// raw_air_data.pitot_temperature
+			// raw_air_data.covariance
+			_raw_air_data_publisher.broadcast(raw_air_data);
+
+			if (hrt_elapsed_time(&_last_diff_press_temperature_publish) > 1_s) {
```

Comment:
I think this part can be removed entirely, it's publishing `uavcan::equipment::air_data::RawAirData` twice.

Reply:
Okay, I changed it to just publish temperature every time through